src/LighthouseServiceProvider.php

```diff
@@ -87,15 +93,36 @@ public function register(): void
         $this->app->alias(GraphQL::class, 'graphql');
 
         $this->app->singleton(DirectiveFactory::class);
-        $this->app->singleton(ExtensionRegistry::class);
         $this->app->singleton(NodeRegistry::class);
         $this->app->singleton(TypeRegistry::class);
         $this->app->singleton(CreatesContext::class, ContextFactory::class);
         $this->app->singleton(CanStreamResponse::class, ResponseStream::class);
-        $this->app->singleton(GraphQLResponse::class, Response::class);
 
-        $this->app->singleton(SchemaSourceProvider::class, function () {
-            return new SchemaStitcher(config('lighthouse.schema.register', ''));
+        $this->app->bind(GraphQLResponse::class, function (): GraphQLResponse {
+            return new class implements GraphQLResponse {
```

Comment:
Why are we creating an anonymous class here? I think we should create a class for this.

Reply:
I figured in this case the implementation is so trivial. Any downsides to the approach i chose? Open to changing it to a class

Reply:
If I want to check which implementation of `GraphQLResponse` it is, then this makes it a lot harder :)